projects/i18n/src/lib/service/message-format-translation-service.spec.ts

```diff
@@ -81,33 +82,58 @@ describe('MessageFormatTranslationService', () => {
         expect(translationService.translate('multiple.params', [{ count: 2, name: 'World' }])).toBe(`many Worlds`);
     });
 
-    it('translates the string via the translation loader via combined translation', () => {
+    it('translates the string via the translation loader via combined translation', done => {
```

Comment:
Why use the done callback? That should be used as a last resource. Can't we just make it async and use await?

Reply:
Weren't there other tests that did the same? We should have them all avoid done since that's the old way of doing things

Reply:
Removed all the usages of `done` from the i18n library. The only place where they still are is in the data exporter specs and I don't think there's a (simple) way to rewrite these without using done.